### PR TITLE
Hosting Metrics: Add two scales on the first line chart

### PR DESCRIPTION
**Pull request by sejas**
- Related to https://github.com/Automattic/dotcom-forge/issues/3274

## Proposed Changes

* Add two scales by defining a `scale` property inside a series.
* The name defined can be anything, in this example is `requests-per-min` but it could be `foo`.
* Not sure if it is possible to align the numbers in each line, and displaying lines per each scale is confusing. Maybe it's better to use different colors. In this PR I just hide the horizontal lines for the new scale.

## Testing Instructions

* Navigate to `/site-monitoring/${siteSlug}`
* Observe the two scales on the first chart

### Screencast

**Before**

https://github.com/Automattic/wp-calypso/assets/779993/c8de6c17-902b-4691-b6cf-f51285cf9db3 

**After**

https://github.com/Automattic/wp-calypso/assets/779993/48dba729-82f3-4b3a-a662-ec4510077d55

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Comment by danielbachhuber** [comment]
@sejas Ahhhhhh!!!!! You are awesome!!!!!

File: client/my-sites/site-monitoring/components/site-monitoring-line-chart/index.tsx [hunk 1]
```diff
@@ -92,6 +93,24 @@ function createSeries( series: Array< SeriesProp > ) {
 	return [ { label: ' ' }, ...configuredSeries ];
 }
 
+function addExtraScaleIfDefined( series: Array< SeriesProp > ) {
+	const scale = series.find( ( serie ) => serie.scale )?.scale;
+	if ( scale ) {
+		return [
```
Comment (wojtekn) [inline client/my-sites/site-monitoring/components/site-monitoring-line-chart/index.tsx:99]: What do you think about using early return here?

**Review by wojtekn — APPROVED** [review]
Great addition, now I realize that two scales for two series charts is a must-have.

File: client/my-sites/site-monitoring/components/site-monitoring-line-chart/index.tsx (hunk 1 above)
Comment (danielbachhuber) [reply client/my-sites/site-monitoring/components/site-monitoring-line-chart/index.tsx:99]: I'm not sure I understand the suggestion. Can you clarify?

Comment (wojtekn) [reply client/my-sites/site-monitoring/components/site-monitoring-line-chart/index.tsx:99]: I think it's cleaner to use an 'early return' approach if possible:

```
if ( ! scale ) {
    return [];
}
// the rest of the code
```

**Review by wojtekn — APPROVED** [review]

**Pull request merged**